### PR TITLE
Add my PPP; fix duplicate recipe bug

### LINKED ISSUE #110: [PE-D] Addr bug: Duplicate Recipes are not detected
Program detects duplicate items but not duplicate recipes

-------------
Labels: `severity.Medium` `type.FunctionalityBug`
original: GabrielSimbingyang/ped#7

### LINKED ISSUE #152: PPP - Rahul

### PR DESCRIPTION
**Pull request by Rahul0506**

### CODE REVIEW

File: docs/team/khenghun.md [hunk 1]
```diff
@@ -12,6 +12,7 @@ which cater for specific games. Users interact with it via a CLI, and receive fe
 with JavaFX.
 
 Given below are my contributions to the project.
+
```
Comment (justacasul) [inline docs/team/khenghun.md:15]: ??????????????

File: docs/team/rahul.md [hunk 2]
```diff
@@ -4,46 +4,69 @@ title: Gidijala Sai Rahul's Project Portfolio Page
 ---
```
Comment (justacasul) [inline docs/team/rahul.md]: rmb to change the file name to your github username

**Review by justacasul — COMMENTED** [review]
bruh
image

File: docs/team/khenghun.md (hunk 1 above)
Comment (Rahul0506) [reply docs/team/khenghun.md:15]: His was the only one without a newline there

**Comment by Rahul0506** [comment]
> bruh

Fixed, thanks for reminding

**Comment by justacasul** [comment]
logotomo

**Pull request merged**